src/plugins/intel_cpu/tests/unit/snippets_transformations/mul_add_to_fma.cpp

```diff
@@ -39,7 +40,7 @@ class EltwiseWithMulAddFunction : public SnippetsFunctionBase {
 
         std::shared_ptr<Node> data2;
         if (scalar_input) {
-            data2 = op::v0::Constant::create(precision, {}, {2.f});
+            data2 = op::v0::Constant::create(precision, {1}, {2.f});
```

Comment:
Why do we change scalar's shape here and below? As I remember, a scalar shape was specified intentionally

Reply:
I think it's an artifact from the future 🤓 
This is likely a piece of shape_inference_infrastructure PR. I will revert this change for now so we'll discuss constant shapes a bit later